__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -172,7 +178,8 @@ describe('TESTING ChatSchema', () => {
         message: {
           created_at: 'createdAt',
           id: 'messageId',
-          updated_at: 'updatedAt'
+          updated_at: 'updatedAt',
+          attachments: '[]'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as line 150-151, ensure to parse the JSON string back into an array when reading from the database.



<!-- This is an auto-generated comment by OSS CodeRabbit -->